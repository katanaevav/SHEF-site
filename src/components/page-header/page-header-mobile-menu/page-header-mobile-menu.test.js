import React from "react";
import renderer from "react-test-renderer";
import PageHeaderMobileMenu from "./page-header-mobile-menu.jsx";

it(`Render PageHeaderMobileMenu`, () => {
  const tree = renderer.create(
      <PageHeaderMobileMenu />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
