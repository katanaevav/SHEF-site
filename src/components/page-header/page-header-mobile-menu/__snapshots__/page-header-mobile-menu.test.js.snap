// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render PageHeaderMobileMenu 1`] = `
<div
  className="page-header__mobile-menu mobile-menu"
>
  <nav
    className="mobile-menu__mobile-nav mobile-nav"
  >
    <ul
      className="mobile-nav__menu "
    >
      <li
        className="mobile-nav__item"
      >
        <a
          href="/catering"
          onClick={[Function]}
          title="Кейтеринг"
        >
          Кейтеринг
        </a>
      </li>
      <li
        className="mobile-nav__item"
      >
        <a
          href="/online-cooking"
          onClick={[Function]}
          title="Бизнесу"
        >
          Онлайн кулинария
        </a>
      </li>
      <li
        className="mobile-nav__item"
      >
        <a
          href="#"
          onClick={[Function]}
          title="Бизнесу"
        >
          Доставка еды
        </a>
      </li>
      <li
        className="mobile-nav__item"
      >
        <a
          href="#"
          onClick={[Function]}
          title="О нас"
        >
          Стань партнером
        </a>
      </li>
    </ul>
  </nav>
  <div
    className="mobile-menu__mobile-apps"
  >
    <a
      className="mobile-menu__mobile-app"
      href="#"
    >
      <img
        alt="Скачать в App Store"
        className="mobile-menu__mobile-app-image"
        height="48"
        src="./img/app-store.png"
        srcSet="./img/app-store@2x.png 2x"
        width="142"
      />
    </a>
    <a
      className="mobile-menu__mobile-app"
      href="#"
    >
      <img
        alt="Скачать в Google Play"
        className="mobile-menu__mobile-app-image"
        height="48"
        src="./img/google-play.png"
        srcSet="./img/google-play@2x.png 2x"
        width="160"
      />
    </a>
  </div>
  <a
    className="mobile-menu__mobile-phone"
    href="tel:+79953014090"
  >
    +7 995 301 40 90
  </a>
  <a
    className="mobile-menu__mobile-jscorp"
    href="#"
  >
    <img
      alt="JSCorp"
      className="mobile-menu__mobile-jscorp-image"
      height="63"
      src="./img/jscorp-logo.svg"
      width="63"
    />
  </a>
</div>
`;
